src/test/java/com/github/tomakehurst/wiremock/jetty11/JettyHttpServerTest.java

```diff
@@ -82,7 +84,7 @@ public void testStopTimeout() {
 
   @Test
   public void testStopTimeoutNotSet() {
-    long expectedStopTimeout = 1000L;
+    long expectedStopTimeout = 5000L;
```

Comment:
Does the original 1s value cause problems with this Jetty verrsion?

Reply:
Yes, this is very interesting but the graceful shutdown in Jetty 11 is taking more time, ~3s on average, I set it to 5s to eliminate possible flakiness.

Reply:
Wow, 3s is pretty bad. Since many (most probably) WireMock tests start and stop the server per test case, this is going hurt performance pretty badly.

I wonder if there's something we do to tune this (or something we're doing to cause in in the first place)?

Reply:
Yeah, we surely could look at it, I have not spend much time into looking but `Graceful.shutdown` is the one taking time, we could look which components here to look after